syntax = "proto3";

package svg_db;


message Rgba {
    uint32 red = 1;
    uint32 green = 2;
    uint32 blue = 3;
    double opacity = 4;
}

message Color {
	uint32 type = 1;
	string valstr = 2;
	Rgba valrgba = 3;
}
message Point {
	double x = 1;
	double y = 2;
}
	